Adyen.xcodeproj/xcshareddata/xcschemes/AdyenSwiftUIHost.xcscheme

```diff
@@ -40,8 +40,8 @@
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "F9615FAE252DDA4800FA8EDD"
-               BuildableName = "AdyenSwiftUITests.xctest"
-               BlueprintName = "AdyenSwiftUITests"
+               BuildableName = "IntegrationSwiftUITests.xctest"
```

Comment:
As far as I know we don't actually execute these tests as they were so unreliable - maybe a good idea to remove them

The only reason we have the SwiftUI target as of right now is to show how to integrate with SiftUI

Reply:
Good to know. I'll play around with these tests and deleted them in the next PR if there is no hope :)